cluster/agent/admin/admin_checks.go

```diff
@@ -235,7 +236,32 @@ func VerifyServiceScheduling() error {
 		return err
 	}
 
-	nwId, err := createNetwork(c, testNetwork)
```

Comment:
This has truly become a frankenstein function, far too complex at this point to merge as is. This needs to be broken down into functions for a serious reduction in cyclomatic complexity.